magento/magento2
# 15963 fix issue - Imported Configurable Products Don't Retain Type Without Children/Configurations

### Description (*)

Imported Configurable Products Don't Retain Type Without Children/Configurations.
### Fixed Issues (if relevant)

1. magento/magento2#15963: Imported Configurable Products Don't Retain Type Without Children/Configurations

### Manual testing scenarios (*)

1. Backend -> Catalog->Product->Add product as configurable product and don't assign configurable product.(It should save as configurable product instead of simple product)
2. Backend -> Store->Import->Set entity type to Product. add csv file with product type configurable product, don't add configurable attribute.(It should save as configurable product instead of simple product)

### Contribution checklist (*)
